.drone.env

```diff
@@ -3,5 +3,5 @@ CORE_COMMITID=b262ada63569e241f9cee451c23f267512fd48b2
 CORE_BRANCH=master
 
 # The test runner source for UI tests
-WEB_COMMITID=41c62d459c4e7bc4a04367eb80f45a8c29aa3baa
+WEB_COMMITID=8420a49515254e17654ea9a32ba611696bf461b6
```

Comment:
Bumping the web commit ID without a corresponding web release can't work since the actual frontend code is being pulled via the linked release in the web makefile

Reply:
done